src-built-in/components/windowTitleBar/src/components/left/LinkerGroups.jsx

```diff
@@ -5,46 +5,59 @@
 import React from "react";
 import { FinsembleHoverDetector } from "@chartiq/finsemble-react-controls";
 import { getStore, Actions as HeaderActions } from "../../stores/windowTitleBarStore";
+import { getChannelLabelFromIndex } from "../../../../shared/linkerUtil";
 let windowTitleBarStore;
+let accessibleLinker = false;
```

Comment:
This line can be deleted

Reply:
@zona-ciq I had to put this line back in (as true). Otherwise I experienced callback errors.